### DISCUSSION TITLE
[Feature Request]: Detach tab to new window

### DISCUSSION THREAD

**Discussion by biplab37** [zen-browser/desktop#2267] [Ideas]
Sometimes it is useful to be able to detach a tab to a new window. In firefox and chrome, one can drag a tab outside to create a new window. Is there a plan to include this feature in zen?

**Comment by gnc-07** [comment]
You can do that already. Zen is based on Firefox.

**Comment by biplab37** [comment]
Unfortunately, it is not working on my computer. Is anyone else having the same problem?
